### PR TITLE
cgroup: Fix python3 TypeError issue

### PR DESCRIPTION
**Pull request by xiangchunfu**
Fix python3 TypeError

id: 1768391, 1766953
Signed-off-by: Xiangchun Fu <xfu@redhat.com>

### CODE REVIEW

**Comment by xiangchunfu** [comment]
(1/1) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (1204.78 s)

File: qemu/tests/cgroup.py [hunk 1]
```diff
@@ -1211,7 +1211,7 @@ def _generate_verification(cpusets, cpus):
             for i in range(len(stats)):
                 matrix.append(['%d' % i])
                 for j in range(len(stats[i])):
-                    if ((stats[i][j] < (verify[i][j] - limit)) or
+                    if ((stats[i][j] < (verify[i][j] - limit - 10)) or
```
Comment (xutian) [inline qemu/tests/cgroup.py]: why downgrade  value here ?

Comment (xiangchunfu) [reply qemu/tests/cgroup.py]: @xutian  stats is physical cpu currently utilization,  verify is physical cpu utilization expect values(100%).  sometimes pyhsical cpu utilization is ~90%.  But I think it makes sense. so  I hope 90%<real utilization <100%.   

This is the latest test result.

verify~stats: 100 ~ 99
verify~stats: 100 ~ 99
verify~stats: 100 ~ 99
.......

Comment (xiangchunfu) [reply qemu/tests/cgroup.py]: @xutian  I dobule checked this case's code again.  All vcpus are pined to physical cpus.  and run infinite loop for per vcpu.

Comment (xiangchunfu) [reply qemu/tests/cgroup.py]: re-run 20 times.  only one time fail.  It is accidental.  so I think don't need to change limition.  

 (01/20) repeat1.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (175.95 s)
 (02/20) repeat2.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (197.38 s)
 (03/20) repeat3.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (173.26 s)
 (04/20) repeat4.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (192.48 s)
 (05/20) repeat5.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (174.12 s)
 (06/20) repeat6.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (173.62 s)
 (07/20) repeat7.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: FAIL: Scenerios 0(0), 0(1), 0(2), 0(3), 0(4), 0(6), 0(7), 1(9), 1(10), 1(11), 1(12), 1(13), 1(14), 1(15),  FAILED (174.76 s)
 (08/20) repeat8.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (170.88 s)
 (09/20) repeat9.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (176.07 s)
 (10/20) repeat10.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (175.43 s)
 (11/20) repeat11.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (173.49 s)
 (12/20) repeat12.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (172.64 s)
 (13/20) repeat13.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (194.59 s)
 (14/20) repeat14.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (176.58 s)
 (15/20) repeat15.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (172.39 s)
 (16/20) repeat16.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (176.65 s)
 (17/20) repeat17.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (172.04 s)
 (18/20) repeat18.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (194.02 s)
 (19/20) repeat19.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (175.69 s)
 (20/20) repeat20.Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.1.x86_64.io-github-autotest-qemu.cgroup.cpuset_cpus: PASS (194.54 s)
RESULTS    : PASS 19 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0

**Comment by xutian** [comment]
Acked-by: Xu Tian <xutian@redhat.com>

**Pull request merged**